### ISSUE TITLE
nl2br filter

### ISSUE THREAD

**Issue by ryanhinkel** [wiota/facade#5]
One nl character results in an escaped <br> instead of a safe one.

**Comment by ryanhinkel** [comment]
Commit 6cd2b16b2c90380ad066afc190302333dcf39f2d addressed this bug. However in the process I had tried to fix the bug with the unicode "u". @di Can you take a quick look [here](https://github.com/wiota/toolbox/blob/master/template_filters.py#L15)

**Comment by di** [comment]
What's "the bug with the unicode 'u'"? I don't see what's wrong here.

**Comment by ryanhinkel** [comment]
Sorry, that wasn't clear. I meant I had tried to fix the bug by adding the unicode u [here](https://github.com/wiota/toolbox/commit/dae33faa874c23459c26238a745627405e96fcf0). Then I found the [real problem](https://github.com/wiota/toolbox/commit/6cd2b16b2c90380ad066afc190302333dcf39f2d). 

Note that I only took out one of the two u's that I put in in the first commit. My question is, what is best practice with the u's because I know I didn't follow it.
